File: files/INSPINIA_v4.2.0/ASP.Net-MVC5/Seed/Controllers/ErrorController.cs
using Microsoft.AspNetCore.Mvc;

namespace Inspinia.Controllers
{
    public class ErrorController : Controller
    {
        public IActionResult Error400() => View();
        public IActionResult Error401() => View();
        public IActionResult Error403() => View();
        public IActionResult Error404() => View();
        public IActionResult Error408() => View();
        public IActionResult Error500() => View();
        public IActionResult Maintenance() => View();
    }
}

File: files/INSPINIA_v4.2.0/RoR/Full/app/controllers/error_controller.rb
class ErrorController < ApplicationController
    layout :choose_layout

    def choose_layout
      action_name == "error_404_alt" ? "vertical" : "base"
    end

    def error_400
    end

    def error_401
    end

    def error_403
    end

    def error_404
    end

    def error_408
    end

    def erro_500
    end

    def maintenance
    end
end
